src/app/(Authentication)/Welcome/index.tsx

```diff
@@ -1,16 +1,22 @@
-import { router } from 'expo-router';
 import React from 'react';
 import { Text, View } from 'react-native';
-import { TouchableOpacity } from 'react-native-gesture-handler';
+import StyledButton from '../../../components/StyledButton';
 import styles from './styles';
 
 function WelcomeScreen() {
   return (
     <View style={styles.container}>
-      <Text>This is the welcome screen!</Text>
-      <TouchableOpacity onPress={() => router.back()} style={styles.button}>
-        <Text>Go Back</Text>
-      </TouchableOpacity>
+      <View style={styles.everythingContainer}>
+        <View style={styles.textContainer}>
+          <Text style={styles.welcomeText}>
+            Welcome to Impact Fund! Log in or create an account to get started.
+          </Text>
+        </View>
+        <View style={styles.buttonContainer}>
+          <StyledButton text="LOG IN" file="/(Authentication)/Login" />
```

Comment:
When the directory is wrapped in parentheses, that means we can ignore it in the path so the file here should be `/Login`